databricks-catalog-external-location/main.tf

```diff
@@ -0,0 +1,169 @@
+data "aws_caller_identity" "current" {
+  provider = aws
+}
+
+locals {
+  path                   = "/databricks/"
+  name                   = "${var.tags.project}-${var.tags.env}"
```

Comment:
I think you'll want to allow folks to pass in a string as well, in case we need to set up more than one external location per workspace

That's already the case for Edu, for example